tests/console/copy_iso_to_external_drive.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Copy the installation ISO to an external drive
+# Maintainer: Joachim Rauch <jrauch@suse.com>
+# Tags: boo#1040749
+
+use base 'btrfs_test';
+use strict;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # choose unpartioned disk and set <$disk> shell variable
+    $self->set_playground_disk;
+    my $DISK = get_var('PLAYGROUNDDISK');
```

Comment:
I'd advise to use lower-case variable names and keep upper-case for test variables.